lessons/misc/ml-intro/kmeans.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: Unsupervised Learning
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+rm(list = ls())
+```
+
+## GitHub Documents
+
+This is an R Markdown format used for publishing markdown documents to GitHub. When you click the **Knit** button all R code chunks are run and a markdown file (.md) suitable for publishing to GitHub is generated.
```

Comment:
This does not appear to have anything to do with Unsupervised Learning.

Reply:
OK, so `kmeans` is mentioned at least once, and I realize some plots are necessary for this, but there's more plotting code than ML code...

Reply:
Yes, you can definitely get rid of lines 10 to 12.